.kokoro/psm_interop_kokoro_lib.sh

```diff
@@ -694,6 +739,10 @@ activate_gke_cluster() {
       GKE_CLUSTER_NAME="interop-test-psm-basic"
       GKE_CLUSTER_ZONE="us-central1-c"
       ;;
+    GKE_CLUSTER_DUAL_STACK)
+          GKE_CLUSTER_NAME="psm-interop-lb-primary-ds"
```

Comment:
Would be great if we used the new cluster name here
```suggestion
          GKE_CLUSTER_NAME="psm-interop-dualstack"
```